src/components/Avatar/Avatar.vue

```diff
@@ -99,6 +99,13 @@ export default {
 		disableTooltip: {
 			type: Boolean,
 			default: false
+		},
+		/**
+		 * Define username is not a user name
```

Comment:
I think we should add a note, that this will disable to load an avatar from the server. The current description is a bit confusing. :wink: